app/models/case_request.rb

```diff
@@ -21,13 +21,15 @@ def initialize(opts = {})
   validate :case_must_exist_on_glimr, if: -> { errors.empty? }
 
   def process!
-    fee_liabilities.each do |fee|
-      prepare_fee(fee)
+    unless glimr_case_request.error?
```

Comment:
This wasn't for mutation testing, but because rubocop was bitching about unverified doubles. When I changed the double to be for the actual class that we get back for this test (CaseNotFound), rubocop pointed out that it doesn't implement fee_liabilities